crates/open_ai/src/open_ai.rs

```diff
@@ -473,7 +473,10 @@ pub async fn stream_completion(
             .filter_map(|line| async move {
                 match line {
                     Ok(line) => {
-                        let line = line.strip_prefix("data: ")?;
+                        let mut line = line.strip_prefix("data:")?;
+                        if line.starts_with(" ") {
+                            line = &line[1..]
+                        }
```

Comment:
```suggestion
                       let line = line.strip_prefix("data: ").or_else(|| line.strip_prefix("data:"))?;
```
What do you think of just doing something like this?